openapi-v1.yaml

```diff
@@ -5487,6 +5591,113 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /organizations/{organization}/sso_domain:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+    post:
+      tags:
+        - organization
+      description: >
+        Create a new SSO connection that connects this organization to this
+        domain.
+      operationId: createSSODomain
+      parameters:
+        - name: config
+          in: body
+          description: The SSO connection to create.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+          required: true
+      responses:
+        200:
+          description: Claim created successfully.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+
+  /organizations/{organization}/sso_domains:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+    get:
+      tags:
+        - organization
+      description: >
+        Lists all the SSO connections associated with the given organization.
+      operationId: getSSODomains
+      responses:
+        200:
+          description: The SSO domains associated with the org.
+          schema:
+            $ref: '#/definitions/SSODomainConfigResponse'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+
+  /organizations/{organization}/sso_domains/{uuid}:
+    parameters:
+      - name: organization
+        in: path
+        description: organization name
+        required: true
+        type: string
+      - name: uuid
+        in: path
+        description: configuration ID
+        required: true
+        type: string
+    get:
+      tags:
+        - organization
+      description: Gets details about the given SSO domain connection.
+      operationId: getSSODomain
+      responses:
+        200:
+          description: The details about this domain connection.
+          schema:
+            $ref: '#/definitions/SSODomainConfig'
+        default:
+          description: error response
+          schema:
+            $ref: '#/definitions/Error'
+    patch:
+      tags:
+        - organization
+      description: Updates the configuration for the given SSO connection.
+      operationId: updateSSODomain
```

Comment:
The PATCH body is missing. Should we add the SSODomainConfig ref or a new definition?

Reply:
that seems important